src/main/java/frc/robot/subsystems/PivotSubsystem.java

```diff
@@ -37,12 +37,13 @@ public PivotSubsystem(){
         m_PidController.setFF(kFF);
         m_PidController.setOutputRange(kMinOutput, kMaxOutput);
 
-        SmartDashboard.putNumber("pivotPOS", 73.08);
+        SmartDashboard.putNumber("pivotPOS", 5);
```

Comment:
I'm guessing you don't need this anymore?

Reply:
I think it would still be good to have, just incase